src/Repository/ClienteRepository.php

```diff
@@ -49,4 +49,13 @@ public function editar($id,$dados): bool
         return True;
     }
 
+    public function buscarClienteCPF($cpf) : ?Cliente
+    {
+        $query = $this->createQueryBuilder("cliente")
+        ->andWhere("cliente.cpf = :cpf")
+        ->setParameter("cpf",$cpf)
+        ->getQuery();
+       return $query->getOneOrNullResult();
+    }
+
```

Comment:
Como o repository já é de Cliente então podemos deixar mais simples o nome do método...

```suggestion
    public function buscarPorCpf($cpf): ?Cliente
    {
        return $this->createQueryBuilder("cliente")
            ->andWhere("cliente.cpf = :cpf")
            ->setParameter("cpf", $cpf)
            ->getQuery()
            ->getOneOrNullResult();
    }
```